### DISCUSSION TITLE
[KubeVela] Enhance multi-cluster observability

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#577] [Google Summer of Code 2022]
Extend the observability of multi-cluster information on KubeVela control plane

Mentor(s): Da Yin (@Somefive)

https://github.com/cncf/mentoring/blob/main/summerofcode/2022.md#enhance-multi-cluster-observability

**Comment by rvacaru** [comment]
Hi KubeVela people, 
I'm Razvan Vacaru, a data engineer based in Amsterdam. I'm looking into the GSoC 2022 ideas and this is one that I think I'll like and I'll be able to help with.

In short my cv is:
- Bachelor's degree in CS Engineering (Java, C, Javascript, Linux sysadmin)
- Worked ~6 years mostly as a Java/Spring developer (4 of the 6 years in AWS)
- Working for ~ 2 years as a Data engineer with python, spark, terraform, datadog, ...

Check the full CV in LinkedIn: https://www.linkedin.com/in/razvan-vacaru-17787b5a/
From time to time I also do small contributions to OS projects like akka or dbt: https://github.com/rvacaru

My goal is to increase my knowledge of K8s and Golang. 
Do you think I would be a good fit for this project? is the fact that we're based in Amsterdam and Hangzhou a problem?
